src/test/java/io/confluent/idesidecar/restapi/resources/KafkaConsumeResourceIT.java

```diff
@@ -496,9 +608,13 @@ void createLocalKafkaTopic(String connectionId,
                 partitionsCount
             )
         )
-        .post(String.format("/kafka/v3/clusters/%s/topics", localKafkaClusterId))
-        .then()
-        .statusCode(201);
+        .post(String.format("/kafka/v3/clusters/%s/topics", localKafkaClusterId));
+    if (response.statusCode() != 201) {
+      System.err.println("Failed to create Kafka topic. Status code: " + response.statusCode());
+      System.err.println("Response body: " + response.getBody().asString());
```

Comment:
Do we need these lines? If so, can we use something other than `System.err` here?

Reply:
Done.